nx/public/sl/components.css

```diff
@@ -152,14 +170,88 @@ button {
     outline-offset: 4px;
   }
 
+  &:not(:disabled) {
+    cursor: pointer;
+  }
+
+  &.negative {
+    background: var(--s2-red-900);
+    border-color: var(--s2-red-900);
+    outline-color: var(--s2-red-900);
+
+    &:not(:disabled):hover {
+      background: var(--s2-red-800);
+      border-color: var(--s2-red-800);
+    }
+  }
+
```

Comment:
I _think_ this is the only thing to keep in this. We're going to make a dedicated read only view, so we shouldn't have to ship so many disabled states for now.

Reply:
Let's clean this up when merging with your updates.